__tests__/app.test.js

```diff
@@ -56,3 +56,35 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: responds with an article object with all these properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(Array.isArray(article)).toBe(true);
+        expect(article).toHaveLength(13);
+        article.forEach((eachArticle) => {
+          expect(eachArticle).toHaveProperty("author");
+          expect(eachArticle).toHaveProperty("title");
+          expect(eachArticle).toHaveProperty("article_id");
+          expect(eachArticle).toHaveProperty("topic");
+          expect(eachArticle).toHaveProperty("created_at");
+          expect(eachArticle).toHaveProperty("votes");
+          expect(eachArticle).toHaveProperty("article_img_url");
+          expect(eachArticle).toHaveProperty("comment_count");
+          expect(article).not.toHaveProperty("body");
+        });
```

Comment:
You should be testing to see if the articles come back in created_at order - descending. You could use the jest-sorted library for this in your test.

Reply:
Added test with jest-sorted